tests/testthat/test-exposure_metrics.R

```diff
@@ -169,3 +169,104 @@ test_that("handles IOV correctly", {
   expect_false(result2a == result2b) # iov is different after day 3
 })
 
+test_that("calc_time_to_target: correct time to target calculated", {
+  regimen <- PKPDsim::new_regimen(
+    amt = 150,
+    n = 20,
+    interval = 12,
+    type = "infusion"
+  )
+
+  trough_target <- create_target_design(
+    targettype = "trough",
+    targetmin = 1,
+    targetmax = 5,
+    at = 6,
+    anchor = "day"
+  )
+  time_to_target_trough <- calc_time_to_target(
+    regimen = regimen,
+    target_design = trough_target,
+    auc_comp = NULL,
+    model = mod_1cmt_iv,
+    covariates = NULL,
+    parameters = list(CL = 5, V = 50)
+  )
+  expect_equal(time_to_target_trough, 13)
```

Comment:
```suggestion
  # attains target on second dosing interval + 1-hr infusion = 13 hr
  expect_equal(time_to_target_trough, 13)
```

very optional suggest but it might be helpful to have some code comments in the test file to explain why we expect a value of e.g. 13 here.

Reply:
My comment above also applied to the other unit test expectations, if you could add some comments there too?